code/game/objects/items/melee/misc.dm

```diff
@@ -1079,3 +903,92 @@
 			target.LAssailant = user
 
 		cooldown = world.time + 15
+
+//Former Wooden Baton
+/obj/item/melee/tonfa
+	name = "Police Tonfa"
+	desc = "A traditional police baton for gaining the submission of an uncooperative target without the use of lethal-force. \
+		As with all traditional weapons, the target will find themselves bruised, but alive. It has proven to be effective in preventing \
+		repeat offenses and has brought employment to lawyers for decades."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "beater"
+	item_state = "beater"
+	worn_icon_state = "classic_baton"
+	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
+	force = 12
+	throwforce = 7
+	slot_flags = ITEM_SLOT_BELT
+	w_class = WEIGHT_CLASS_LARGE
+	hitsound = 'sound/effects/woodhit.ogg'
+	/// Damage dealt while on help intent
+	var/non_harm_force = 4
+	/// Stamina damage dealt
+	var/stamina_force = 25
+
+/obj/item/melee/tonfa/attack(mob/living/target, mob/living/user)
+	var/target_zone = user.get_combat_bodyzone(target)
+	var/armour_level = target.getarmor(target_zone, STAMINA, penetration = armour_penetration - 15)
```

Comment:
Hmm, I am not sure, but I wonder if these should stay here, or moved to below since `target_zone` would be only available from carbon mobs... but `armour_level`'s proc wants to have a target_zone...